sql/tools/toMongo.js

```diff
@@ -0,0 +1,36 @@
+import db from '../sequelize';
+import { User as MongoUser } from '../../server/models/user.model';
+
+const SqlUser = db.User;
+const SqlDevice = db.Device;
+
+(async () => {
+    const LIMIT = 1000;
+    let offset = 0;
+    let users = [];
+    do {
+        users = (await SqlUser.findAll({
+            offset,
+            limit: LIMIT,
+            include: [{
+                model: SqlDevice,
+            }],
+        })).map(user => ({
```

Comment:
I'm doing a small transformation in javascript here. There's probably a way to do most of this in SQL but I'm no SQL expert and figured the productivity vs performance tradeoff was okay in this case.

If we think we should do more of this in SQL, I'll need some help :)

Reply:
I think this is ok.